src/LoadBalance/LB_Refine_GetNewRealPatchList.cpp

```diff
@@ -158,6 +162,12 @@ void LB_Refine_GetNewRealPatchList( const int FaLv, int &NNew_Home, int *&NewPID
          }
 #        endif // #ifdef MHD
 
+#        if ( MODEL == ELBDM && ELBDM_SCHEME == ELBDM_HYBRID )
+         if ( !amr->use_wave_flag[ SonLv ] && !amr->use_wave_flag[ FaLv ] && TP->use_wave_flag ) {
```

Comment:
Is the check `!amr->use_wave_flag[ FaLv ]` necessary?

Reply:
@hyschive No, I have removed it.